app/src/main/java/org/eyeseetea/malariacare/presentation/presenters/MediaPresenter.java

```diff
@@ -47,14 +48,14 @@ public void onError() {
     }
 
     private void showMediaItems() {
-        if (settings.getMediaListMode().equals(Settings.MediaListMode.LIST)) {
```

Comment:
We should use if (mView != null)  conditions as in other methods in presenter when we communicate with the view to avoid memory leaks